ai/src/data.py

```diff
@@ -182,25 +182,29 @@ def get_current_window(self):
                 if self.get_window_span() == 1 else
                 self.massive_table_filled.iloc[start_index:end_index])
 
-    def get_latest_window(self):
+    def get_window_at(self, time):
```

Comment:
Couldn't we have `time=None` as default value for the latest_window, so we do not need branches latter in the GetInferenceHandler?

Reply:
Yeah, that is a good point. I removed `get_latest_window` entirely.